api/IDBVersionChangeEvent.json

```
@@ -68,6 +68,56 @@
           "deprecated": false
         }
       },
+      "IDBVersionChangeEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBVersionChangeEvent/IDBVersionChangeEvent",
+          "spec_url": "https://w3c.github.io/IndexedDB/#dom-idbversionchangeevent-idbversionchangeevent",
+          "description": "<code>IDBVersionChangeEvent()</code> constructor",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "25"
+            },
+            "firefox_android": {
+              "version_added": "25"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": true
```

```suggestion
              "version_added": "10"
```

Suggested directly by https://github.com/foolip/mdn-bcd-collector#updating-bcd-using-the-results since we have results for Safari 9.1 and 10 in https://github.com/foolip/mdn-bcd-results.